source/assets/stylesheets/sections/_code_of_conduct.scss

```diff
@@ -0,0 +1,7 @@
+body.code-of-conduct {
+  // TODO: Use these sizes for the whole website
+  @media (max-width: $screen-xs-max) { font-size: 16px; }
```

Comment:
Agreed. This one is a work in progress to test the viability and usefulness of responsive font sizes. If it's conclusive, let's open a new issue to tackle and polish this idea for the whole web site.